index.bs

```diff
@@ -2381,17 +2381,15 @@ To <dfn>get the browsing context info</dfn> given |context|,
 </div>
 
 <div algorithm>
-To <dfn>get the descendent browsing contexts</dfn> given |parent id|, |depth|
+To <dfn>get the descendent browsing contexts</dfn> given |parent|, |depth|
 and |max depth|:
 
-1. If |max depth| is greater than zero, and |depth| is equal to |max depth|,
-   return null.
+1. Assert: |parent| is not null.
 
-1. Let |parent| be the result of [=trying=] to [=get a browsing context=]
-   given |parent id|.
+1. If |max depth| is not null, and |depth| is greater than |max depth|,
```

Comment:
I think in the current formulation there's an off-by-one error here, right? If depth is 0 and max depth is 0 we just want to include the parent context, so we should return.

Reply:
Oh I guess there isn't because we only call this from `get the browsing context info`, which increments the depth counter.